File: PYTHON/Repetitions.py
s = str(input())

l = 0
count = 0
res = 0
for r in range(len(s)):
    if s[r] == s[l]:
        count+=1
    else:
        l=r
        res = max(res,count) 
        count = 1
res = max(res,count)
print(res)        
        

File: CPP/Repetitions.cpp
#include <bits/stdc++.h>
using namespace std;



int main()
{
   string n;
   cin >> n;
   int l = 0;
   int count = 0;
   int sol = 1;
   for(int r = 0;r < n.size();r++){
       if(n[l] == n[r]){
           count++;
       }
       else{
           l = r;
           sol = max(sol,count);
           count = 1;
           
       }
   }
   sol = max(sol,count);
   cout << sol ;
  
    return 0;
}